bbn-test-5/babylon-node/README.md

```diff
@@ -98,10 +98,10 @@ Parameters:
 - `minimum-gas-prices`: The minimum gas price (in this example we use `0.005ubbn`)
    that your node will accept for transactions. Transactions with lower gas 
    prices will be rejected.
-- `iavl-cache-size`: Default is "5000" (uses ~3GB of memory). Setting to 
+- `iavl-cache-size`: Default is "781250". Setting to 
    0 disables the IAVL tree caching, which reduces memory usage but significantly 
```

Comment:
Why do we recommend people to have a default with lower performance?

Reply:
Enabling the IAVL cache will exhaust the available memory, even on 128G machines. We need to dig deeper to understand why.

Until then, we decided to turn off the IAVL cache - it won't have a performance impact to regular users.